### PR TITLE
Update ballotexample.json with abstain accts

### PR DESCRIPTION
**Pull request by jimscarver**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
             <img alt="RChain Community" src="https://avatars1.githubusercontent.com/u/43198800?s=88&v=4">
           </a>
         </div>
-        <div class="progress" style="width: 50%">
+        <div class="progress" style="width: 20%">
```
Comment (dckc) [inline public/index.html]: what does width have to do with ballotexample.json and abstain accts?

**Comment by dckc** [comment]
10 commits? are you having trouble? I guess we can just squash them.

**Review by dckc — CHANGES_REQUESTED** [review]
please get rid of the changes that are not relevant to abstain

File: public/index.html (hunk 1 above)
Comment (jimscarver) [reply public/index.html]: this was an attempt to fix screw up on narrow screen covering heading.

Comment (dckc) [reply public/index.html]: I don't see how your answer is responsive to my question, so I re-iterate: what does width have to do with this pull request, which is about ballotexample.json and abstain accts?

**Pull request merged**